robowflex_library/yaml/test_cob4.yml

```diff
@@ -0,0 +1,41 @@
+name: (noname)
+robot_model_name: cob4
+robot_state:
+  joint_state: 
+    position: [0, 0, 0.05, 0, 0, 0, 1.32, 1.4, -0.2, 1.72, 0.0, 1.66, 0.0, 0.04, 0.04]    
```

Comment:
The size of this vector is different from the size of the joints (one is 17 one is 15)

Reply:
Done!